rocketallimages.php

```diff
@@ -37,7 +37,11 @@ public function install()
 
     public function uninstall()
     {
-        return parent::uninstall();
+        if (!parent::uninstall() ||
```

Comment:
maybe just 
`return parent::uninstall && Configuration::deleteByName('ROCKET_ALLIMAGES_ORDERED');`

Reply:
Thank's for review :)